Formula/kopia.rb

```diff
@@ -0,0 +1,49 @@
+class Kopia < Formula
+  desc "Fast and secure open-source backup"
+  homepage "https://kopia.io"
+  url "https://github.com/kopia/kopia.git",
+      tag:      "v0.10.4",
+      revision: "3447f1865f0b0536653aa6e9c07896ce73c3e600"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    # do not set github.com/kopia/kopia/repo.BuildGitHubRepo to disable
+    # update notifications
```

Comment:
Let's define it like this, it's a little clearer
```suggestion
    # removed github.com/kopia/kopia/repo.BuildGitHubRepo to disable
    # update notifications
```